package.json

```diff
@@ -14,11 +14,11 @@
     "build": "webpack --quiet --display-error-details --bail",
     "build:dev": "NODE_ENV=development npm run build",
     "build:test": "NODE_ENV=test npm run build && npm run mock-server:build",
-    "prebuild": "npm run clean && npm run mkdirs && npm run license-checker && npm run check && npm run test",
+    "prebuild": "npm run clean && npm run mkdirs",
     "postbuild": "dts-bundle-generator src/types/index.ts --out-file lib/index.d.ts",
     "mkdirs": "mkdir -p dist && mkdir -p lib",
     "clean": "rimraf lib dist",
-    "prepare": "npm run clean && npm run check && npm run build",
+    "prepare": "npm run clean",
```

Comment:
Why don't we need `... && npm run check && npm run build` anymore?

Reply:
These scripts have been changed only temporarily. I'm going to revert those changes for now but I think it would be good to separate testing from the build.